src/styles/watch.scss

```
@@ -0,0 +1,70 @@
+.watch {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: #{$watch-size};
+  height: #{$watch-size};
+  border-radius: 50%;
+  background-color: rgb(195, 244, 244);
+  border: 4px solid #7ebfee;
+}
+
+.watch__hand {
+  position: absolute;
+  left: 50%;
+  transform-origin: bottom center;
+  border-radius: (50% 50% 0 0);
+
+  &--minute-hand {
+    top: #{($watch-size - 2 * $minute-hand-height) / 2};
+    left: #{($watch-size - 2 * $minute-hand-width) / 2};
+    height: #{$minute-hand-height};
+    width: #{$minute-hand-width};
+    background-color: red;
+    animation-duration: 3600s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    animation-name: minute;
+  }
+
+  &--seconds-hand {
+    top: #{($watch-size - 2 * $second-hand-height) / 2};
+    left: #{($watch-size - 2 * $second-hand-width) / 2};
+    height: #{$second-hand-height};
+    width: #{$second-hand-width};
+    background-color: blue;
+    animation-duration: 60s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    animation-name: second;
+  }
+}
+
+@keyframes minute {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes second {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

its the same animation
```suggestion
@keyframes handRotate {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
```